src/main/scala/intellij/haskell/editor/HaskellCompletionContributor.scala

```diff
@@ -392,6 +402,43 @@ class HaskellCompletionContributor extends CompletionContributor {
     waitForResult(lookupElements)
   }
 
+  private def getIdsFromAllExportedModules(project: Project) = {
```

Comment:
OK, I will load current file as I did before, but what's the problem with `getIdsFromAllExportedModules`, because all exported identifiers can be accessed in any context if we start a stack repl process.